src/main/java/seedu/address/logic/commands/CreatePastAppointmentCommand.java

```diff
@@ -0,0 +1,58 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_DATE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_DIAGNOSIS;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_MEDICATION;
+
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.appointment.PastAppointment;
+import seedu.address.model.person.Person;
+
+/**
+ * Creates a past appointment for a patient.
+ */
+public class CreatePastAppointmentCommand extends Command {
+    public static final String COMMAND_WORD = "appt";
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Creates a past appointment for a patient.\n"
+            + "Parameters: INDEX (Must be a positive integer) "
+            + PREFIX_DATE + "DATE (Must follow dd-mm-yyyy syntax) "
+            + PREFIX_MEDICATION + "MEDICATION "
+            + PREFIX_DIAGNOSIS + "DIAGNOSIS \n"
+            + "Example: " + COMMAND_WORD + " 1 "
+            + PREFIX_DATE + "01-01-2001 "
+            + PREFIX_MEDICATION + "ibuprofen "
+            + PREFIX_DIAGNOSIS + "headache, medicine given for 3 days ";
```

Comment:
Parameter and Example of date format has a discrepancy with the internals of how you parse the LocalDate in your `CreatePastAppointmentCommandParser` class. (Example given was dd-MM-yyyy, but parser only allows ddMMyy format).

Reply:
Fixed, thanks for noticing!